test/com/google/enterprise/adaptor/database/ResponseGeneratorTest.java

```diff
@@ -460,15 +334,18 @@ public void testUrlColumnModeIncorrectColumnName() throws Exception {
         Response.class.getClassLoader(),
         new Class[] { Response.class }, far);
     Map<String, String> cfg = new TreeMap<String, String>();
-    cfg.put("columnName", "my-url-went-that-way");
+    cfg.put("columnName", "wrongcolumn");
     File testFile = null;
     try {
       testFile = File.createTempFile("db.rg.test", ".txt");
       String content = "from a yellow url connection comes monty python";
       writeDataToFile(testFile, content);
+
+      executeUpdate("create table data(url varchar)");
+      executeUpdate("insert into data(url) values ('some URL')");
+
```

Comment:
Ah, good catch. Yes, I started inserting the real URI into the database, but realized that the test shouldn't try to read the real URL column, so making it an invalid URI seemed good. The test is good, the file is just redundant, so let me take a rain check and include this as part of the followup cleanup of the temp files.